indico/modules/events/persons/controllers.py

```diff
@@ -44,6 +47,16 @@
 from indico.web.forms.base import FormDefaults
 from indico.web.util import jsonify_form, jsonify_template
 
+BUILTIN_ROLES = {'chairperson': {'name': 'Chairperson', 'code': 'CHR', 'color': 'f7b076',
+                                 'css': 'color: #f7b076 !important; border-color: #f7b076 !important'},
+                 'author': {'name': 'Author', 'code': 'AUT', 'color': '7994f7',
+                            'css': 'color: #7994f7 !important; border-color: #7994f7 !important'},
+                 'convener': {'name': 'Convener', 'code': 'CON', 'color': 'dcb1f9',
+                              'css': 'color: #dcb1f9 !important; border-color: #dcb1f9 !important'},
+                 'speaker': {'name': 'Speaker', 'code': 'SPK', 'color': '7fd1c8',
+                             'css': 'color: #7fd1c8 !important; border-color: #7fd1c8 !important'}
+                 }
```

Comment:
I'd either move this into the previous line or use this style:

```
BUILTIN_ROLES = {
    'chairperson': ...,
    'author': ...,
}
```

(whatever style you prefer or is more readable, but the indented `}` in a separate line is super ugly)